libr/bin/format/pyc/bn.h

```diff
@@ -0,0 +1,113 @@
+#ifndef __BIGNUM_H__
+#define __BIGNUM_H__
+/*
+
+Big number library - arithmetic on multiple-precision unsigned integers.
+
+This library is an implementation of arithmetic on arbitrarily large integers.
+
```

Comment:
This should be moved to r_util. We have a bignum implementation there already

Reply:
If this bignum impl is moved in t_util, whether it's merged with existing impl or lived standalone?
Cam I just modify the existing big num impl?